### ISSUE TITLE
We need comparation results.  

### ISSUE THREAD

**Issue by yuedajiong** [AdityaNG/kan-gpt#6]

**Comment by aiva00** [comment]
It is in the author's todo list. Any ideas on the comparison options?

**Comment by PriNova** [comment]
Maybe some perplexity metrics against the base GTP-2 model?

**Comment by AdityaNG** [comment]
Hi all, we have added results in the README comparing the KAN-GPT to the MLP-GPT. These are preliminary results and we are running large hyperparameter sweeps. For now, we observe that the KAN-GPT performs slightly better than the MLP-GPT as shown below:

![results](https://github.com/AdityaNG/kan-gpt/assets/8216281/9b9c20be-9d07-45d9-ac1b-14e893b53f58)


README Results: https://github.com/AdityaNG/kan-gpt/?tab=readme-ov-file#results


We are currently running an extensive hyperparameter sweep as shown below:

![image](https://github.com/AdityaNG/kan-gpt/assets/8216281/53e69d13-8ecb-4d57-89a6-065ca5c6d055)

**Comment by AdityaNG** [comment]
We have added the metrics of cross entropy, perplexity along with the loss curves to the [docs](https://adityang.github.io/kan-gpt/results/) and the [README](https://adityang.github.io/kan-gpt/results/)

**Comment by yuedajiong** [comment]
Greeeeeeeeaaaaaat !!!
